src/ResourceManager/Websites/Commands.Websites.Test/ScenarioTests/WebAppTests.cs

```diff
@@ -45,6 +45,13 @@ public void TestCreateNewWebAppHyperV()
             WebsitesController.NewInstance.RunPsTest(_logger, "Test-CreateNewWebAppHyperV");
         }
 
+        [Fact]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void TestEnableContainerContiniousDeploymentAndGetUrl()
```

Comment:
Typo in name. Should be `TestEnableContainer`**Continuous**`DeploymentAndGetUrl`.
You'll also need to rename the .json file too to be correct.

Reply:
Done.